### PR TITLE
INTMDB-509 alert configuration

### PR DESCRIPTION
**Pull request by rdharmedran**
## Description

alert configuration  resource implementation for AWS CFN
Link to any related issue(s): 

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: cfn-resources/alert-configuration/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+cfn invoke DELETE test.request.json 
+```
+
+Both CREATE & DELETE tests must pass.
+
+## Installation
+TAGS=logging make
+cfn submit --verbose --set-default
+
+## Cloudformation Examples
```
Comment (Zuhairahmed) [inline cfn-resources/alert-configuration/README.md:35]:
```suggestion
## CloudFormation Examples
```

File: cfn-resources/alert-configuration/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+Please see the [CFN Template](test/alert-configuration.sample-template.yaml) for example resource
```
Comment (Zuhairahmed) [inline cfn-resources/alert-configuration/README.md:37]: Can you add this template to example folder and link to there instead? 
https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/examples

Comment (rdharmedran) [reply cfn-resources/alert-configuration/README.md]: @Zuhairahmed - We have fixed the review comments.Please review it

**Review by Zuhairahmed — APPROVED** [review]
LGTM!

**Review by matt-condon — APPROVED** [review]
Nice comprehensive null handling. LGTM

**Review by ParthasarathyVaradhan — APPROVED** [review]
LGTM

**Pull request merged**